NGTrackForce/src/app/components/batch-list/batch-list.component.html

```diff
@@ -51,13 +51,13 @@ <h3>Select Dates</h3>
       </form> -->
 
        <!--1807:Test button for conversion from datepicker to html input -->
-       <div class="col-lg-7 col-md-12 mcolumn right">
+       <div class="col-lg-6 col-md-12 mcolumn left">
         <h3>Select Dates</h3>
         <!-- <div *ngIf="showDateRangeError" class="alert alert-danger" role="alert">
             {{dateRangeMessage}}
         </div> -->
       <!--Test Conversion to html date input only, will update upon changing the dates-->
-      <div class="col-lg-7 col-md-12 mcolumn right">
+      <!-- <div class="col-lg-7 col-md-12 mcolumn right"> -->
         <!-- <table class="table borderless fixedw-table nopadding-table">
```

Comment:
Can we take out these commented out pieces of code, or is there a known reason for leaving them?

Reply:
Because the original code has lots of comments so I figured maybe they left it because maybe they were thinking of un commenting them later so I though if those commented line could make a difference so may my commented line can too since it was in there code